### DISCUSSION TITLE
Pin Favorite documents to Elementor's Dashboard widget

### DISCUSSION THREAD

**Discussion by AudioMartini** [elementor/elementor#17754] [Feature Request]
While Elementor features a "Recently Edited" section in the Dashboard, it would be a real timesaver to have a "Favorites" section available to populate as we see fit. We could include specific Pages or specific Posts or specific Custom Posts or whatever specific item, and access them right from the Dashboard. No more navigating through multiple sections with multiple clicks to get to that one Page (or whatever item).

**Comment by nicholaszein** [comment]
You can always use the [Elementor Finder](https://elementor.com/help/the-finder/)!

**Comment by AudioMartini** [comment]
I didn't realize that was a thing until I ran across an article about a year and a half ago! You are correct, that's what I use all the time. Thanks for the suggestion!
